library/nameops.mli

```diff
@@ -50,16 +50,57 @@ val increment_subscript : Id.t -> Id.t
 
 val forget_subscript    : Id.t -> Id.t
 
+module Name : sig
+
+  include module type of struct include Names.Name end
+
+  val fold_right : (Id.t -> 'a -> 'a) -> Name.t -> 'a -> 'a
+  (** [fold_left f a x] is [f a id] if [x] is [Name id], and [a] otherwise. *)
```

Comment:
I don't know. I can see situation where it might be, and other where it does not. 

At least, a quick grep shows that the usage in the code is more not to have comments on `val`'s of `ml` files. So, shall we add this recommendation to the Coq coding style?